src/library_wasi.js

```diff
@@ -181,7 +183,7 @@ var WasiLibrary = {
 #if SYSCALLS_REQUIRE_FILESYSTEM == 0 && (!MINIMAL_RUNTIME || EXIT_RUNTIME)
   $flush_NO_FILESYSTEM: function() {
     // flush anything remaining in the buffers during shutdown
-    if (typeof _fflush !== 'undefined') _fflush(0);
+    if (typeof _fflush !== 'undefined') _fflush({{{ sizeT(0) }}});
```

Comment:
Do you have `pointerT` already?   (fflush takes a pointer)   (if not its probably not worth adding it for this one place).